src/db/params.rs

```diff
@@ -2,7 +2,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, you can obtain one at https://mozilla.org/MPL/2.0/.
 
-//! Data definitions.
+//! Parameter types for database methods.
```

Comment:
I mentioned this in IRC yesterday, but having the input and output type definitions in separate modules just made it feel a bit easier to keep track of things in my head.

Reply:
(another benefit is that it also makes the consuming code more uniform, which improves the refactorability of some of the boilerplate, e.g. everything can just use `params::$type` and `results::$type`)